xarray/core/dataset.py

```diff
@@ -2328,7 +2328,7 @@ def swap_dims(self, dims_dict, inplace=None):
         return self._replace_with_new_dims(variables, coord_names,
                                            indexes=indexes, inplace=inplace)
 
-    def expand_dims(self, dim, axis=None):
+    def expand_dims(self, dim=None, axis=None, **new_dims):
```

Comment:
It would be nice to follow an argument naming convention,
`expand_dims(self, dim=None, axis=None, **dim_kwargs):`